src/react-apps/applications/shared/src/utils/networking.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
+
+export interface IGetRequestResponse {
+  body: any;
+}
+
+export async function get(url: string, options?: any): Promise<any> {
+  try {
+    const response: AxiosResponse = await axios.get(
+      url,
+      options ? options : null,
+    );
+    return response.data ? response.data : null;
+  } catch (err) {
+    throw err;
+  }
+}
+
+export async function post(
+  url: string,
+  options?: AxiosRequestConfig,
+): Promise<void> {
+  try {
+    await axios.post(url, options ? options : null);
+  } catch (err) {
+    throw err;
+  }
+}
+
+export async function put(
+  url: string,
+  apiMode: string,
+  data: any,
+  config?: AxiosRequestConfig,
+): Promise<void> {
+  try {
+    await axios.put(url + `/${apiMode}`, data, config ? config : null);
```

Comment:
The `/${apiMode}` should be in the url you call this function with. Shouldn't have to add it here.

Reply:
:+1: 
We might be PUT-ing into other APIs in the future. The ones who implement that should change and move this to shared folder.